City-of-Helsinki/berth-reservations-admin
# VEN-717 | Winter storage areas page: Detailed row

## Description :sparkles:

* Add detailed row to winter storage areas list

### Related changes ⛓️

* Move `HarborDetails` terms into `common.terminology`
* Rename `HarborMapLinks` to `MapLinks`
* Move `HarborCard` `harborPlaceholder` to common `placeholderImage`, use in `HarborDetails`

### Code quality improvements 🧹

* Adjust linting settings to fix all icon related lint warnings
* Fix all remaining lint warnings

## Issues :bug:

### Closes :no_good_woman:

* [VEN-717](https://helsinkisolutionoffice.atlassian.net/browse/VEN-717): FE - Winter storage areas page: Detailed row

## Testing :alembic:

### Automated tests :gear:️

* Created snapshot test for WinterStorageAreaDetails
* Updated test data and snapshots for winterStorageAreaList utils test

### Manual testing :construction_worker_man:

* Check that the new component is visible and looks as expected
* Check Storybook stories for WinterStorageAreaList and WinterStorageAreaDetails
